### PR TITLE
Content service: added Create, UploadUrl, publish, unlisted publish Apis 

### PR DESCRIPTION
**Pull request by pritha-tarento**

### CODE REVIEW

File: content-service/content-api/app/controllers/ContentV3Controller.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ class ContentV3Controller extends BaseController {
 
     def create():Action[AnyContent] = Action.async {
         implicit request =>
-            val body: String = JSONUtils.serialize(request.body.asJson.getOrElse(""))
+            val body: String = Json.stringify(request.body.asJson.get)//JSONUtils.serialize(request.body.asJson.getOrElse(""))
```
Comment (krgauraw) [inline content-service/content-api/app/controllers/ContentV3Controller.scala]: Please remove commented code.

Comment (pritha-tarento) [reply content-service/content-api/app/controllers/ContentV3Controller.scala]: removed

File: content-service/content-api-core/src/main/scala/org/ekstep/mgr/impl/BaseContentManager.scala [hunk 2]
```diff
@@ -525,7 +527,7 @@ import scala.collection.mutable.MutableList
   }
 
 
-  protected def checkYoutubeLicense(artifactUrl: String, node: Node): Unit = {
+/*  protected def checkYoutubeLicense(artifactUrl: String, node: Node): Unit = {
```
Comment (krgauraw) [inline content-service/content-api-core/src/main/scala/org/ekstep/mgr/impl/BaseContentManager.scala]: Please remove commented code.

Comment (pritha-tarento) [reply content-service/content-api-core/src/main/scala/org/ekstep/mgr/impl/BaseContentManager.scala]: Removed

**Pull request merged**